src/client/components/three/stories/three.stories.tsx

```diff
@@ -139,7 +140,7 @@ class BoxViewModel {
     return mesh
   }
 
-  @computed
+  @disposableComputed
   private get material(): Material {
     return new MeshPhongMaterial({ color: this.model.color })
```

Comment:
Does this still recompile the shader every time you change the colour?

Reply:
Yes 😭

Reply:
And once again all of your wonderful abstractions must die in service of performance :P

Reply:
JUST DON'T CHANGE COLOURS OKAY EVERYONE 😭

Reply:
JUST DON'T DO ANYTHING USEFUL 😭

Reply:
I think we can solve this, but it might be tricky